src/main/java/io/fabric8/maven/docker/config/BuildImageConfiguration.java

```diff
@@ -175,16 +177,19 @@ public AssemblyConfiguration getAssemblyConfiguration() {
         return assembly;
     }
 
+    @Nonnull
     public List<String> getPorts() {
-        return ports;
+        return EnvUtil.removeEmpties(ports);
```

Comment:
Ah. ok. I thought it was about null entries. We should keep it of course like this, maybe calling it 'removeEmptyEntries()' ?

Trim is find for me, too.